index.js

```diff
@@ -108,6 +127,31 @@ function outputChangelog (argv, cb) {
   })
 }
 
+function getNextVersion (release) {
+  var newVersion = semver.inc(pkg.version, release.releaseAs)
+
+  if (argv.preRelease) {
```

Comment:
I think I did this in a previous commit. But then no new minor/major version is created if feat/ breacking change comes. If you find a better way for this problem I'm open for it!